config/plugins/visualizations/media_player/audio_player/config/audio_player.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE visualization SYSTEM "../../visualization.dtd">
-<visualization name="Audio player">
+<visualization name="Audio player" embeddable="false">
```

Comment:
why do you think audio and video should not be playable in a report?

Reply:
I wanted reports to be printable and exportable e.g. as PDF but maybe that is too restrictive. Also I noticed in a test that the video player might have an issue with playing (some?) ".mp4" files.

Reply:
I enabled them for now, maybe we can revisit the audio player and video player in a paper cut issue #10829.

Reply:
> I wanted reports to be printable and exportable e.g. as PDF but maybe that is too restrictive. Also I noticed in a test that the video player might have an issue with playing (some?) .mp4 files.

I think that is a good aim, but we should not restrict more advanced use-cases. We could offer a warning or better fallback to an image taking from a random/first frame.